src/ServiceControl.MultiInstance.AcceptanceTests/ServiceControl.runsettings

```diff
@@ -1,7 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<RunSettings>
-  <RunConfiguration>
-    <!-- The tests need to be run in x64 otherwise an exception about RavenDB (Voron) not being supported in 32bit mode will be thrown. -->
-    <TargetPlatform>x64</TargetPlatform>
```

Comment:
I have verified and I can still run tests locally so this change seems to be :+1: